get-started/setup-an-ide.md

```diff
@@ -1,7 +1,89 @@
 # Setup an IDE
 
-* vs code
-* goland
-* vim \(optional\)
-* emacs \(optional\)
+## vs code
 
+## GoLand
+
+You can use GoLand to easily run or debug TiDB in many situations.
+
+> Goland uses [delve](https://github.com/go-delve/delve) to debug go programs, usually GoLand has an usable delve and needn't any additional configuration. But in some cases, you may need to download a delve by yourself.
```

Comment:
before this note, could you add some reproducible steps about how to install goland in the local machine?

Reply:
maybe a page https://www.jetbrains.com/go/download/ is enough...